feature/feature.go

```diff
@@ -28,9 +28,20 @@ const (
 	// // alpha: v1.X
 	// MyFeature featuregate.Feature = "MyFeature"
 
+	// EKS is used to enable EKS support
 	// owner: @richardcase
 	// alpha: v0.4
 	EKS featuregate.Feature = "EKS"
+
+	// EKSDisableIAM will disable all IAM resource creation/modification if set to true
+	// owner: @richardcase
+	// alpha: v0.4
+	EKSDisableIAM featuregate.Feature = "EKSDisableIAM"
```

Comment:
As this is security impacting, this should be an opt-in feature, not an opt out.

Reply:
Very good point @randomvariable - will get that changed now......